components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,185 @@
+                      <Span paddingTop="10px">
+                        <NextIllustration
+                          width={12}
+                          height={12}
+                          src="/static/images/news-and-updates-ellipse.svg"
+                          alt="News and Updates Ellipse"
+                        />
+                      </Span>
+                      <Text fontSize="20px" margin="0px 12px">
+                        {update.title}
+                      </Text>
+                    </Flex>
+                    <Flex pt={2} pb={3}>
+                      <StyledLink
+                        onClick={event => goToUpdatePage(router, `/opencollective/updates/${update.slug}`, event)}
+                        color="#1153D6"
+                        fontSize="14px"
+                        display="flex"
+                      >
+                        <FormattedMessage
+                          id="NewsAndUpdates.link.giveFeedback"
+                          defaultMessage="Read more & give Feedback"
+                        />
+                      </StyledLink>
+                    </Flex>
+                    <Flex pb={1}>
+                      <HTMLContent color="#313233" mt={1} fontSize="16px" content={update.summary} />
+                    </Flex>
+                    <Flex pb={3}>
+                      {update.summary.slice(update.summary.length - 3) === '...' && (
+                        <StyledLink
+                          onClick={event => goToUpdatePage(router, `/opencollective/updates/${update.slug}`, event)}
+                          fontSize="14px"
+                          display="flex"
+                        >
+                          <FormattedMessage id="NewsAndUpdates.link.readMore" defaultMessage="Read more" />
+                        </StyledLink>
+                      )}
+                    </Flex>
+                  </Container>
+                ))}
+              </StyledCarousel>
+            ) : null
+          }
+        </Query>
+      </ModalBody>
+      <ModalFooter isFullWidth>
+        <Container display="flex">
+          <Flex pt={1} width={1 / 2}>
+            <StyledLink href="https://twitter.com/opencollect" openInNewTab color="#4E5052" display="flex" pt={3}>
+              <Span pr={2}>
+                <FormattedMessage id="NewsAndUpdates.link.twitterFollow" defaultMessage="Follow us" />
+              </Span>
+              <Span pt={0.5}>
+                <NextIllustration
+                  width={16}
+                  height={14}
+                  src="/static/images/twitter-icon-blue.svg"
+                  alt="Updates and News Icon"
+                />
+              </Span>
+            </StyledLink>
+          </Flex>
+          <Flex width={1 / 2} justifyContent="right">
+            <StyledButton
+              onClick={event => {
+                onClose();
+                goToUpdatePage(router, '/opencollective/updates', event);
```

Comment:
Can we rather use a standard `Link`? They're usually better in terms of accessibility than a button + `pushRoute`

Reply:
Done. 😄